### DISCUSSION TITLE
Where's the "PNG Info" tab?

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/sdnext#967] [Q&A] [answered]
I can't see it in this version, is there any way to display it?

**Comment by vladmandic** [comment, marked as answer]
ahh, haven't seen this question for a while - but in early days, it was asked about 30 times so far. i suggest a quick search next time.
its integrated into process image. just load/drop image there and you have full functionality.

**Reply by ghost** [reply]
I clicked on search for similar questions, nothing was displayed. Thanks for your answer, although the button "Send to txt2img" isn't there like in the original A1111's GUI if I'm not mistaken.

**Reply by vladmandic** [reply]
yes it is, just below the image once you load the image.

**Reply by ghost** [reply]
Right, that button wasn't working until I completely restarted the programme so that's why I thought there was no such function... 😅

**Comment by FullBleed** [comment]
Maybe add an in-app FAQ.  ;)

**Reply by ghost** [reply]
Yeah, the truth is that I'm quite used to the original A1111's and switching to this one is quite a radical change and I'm a bit lost.
